### DISCUSSION TITLE
Find zip file generated during "sf project deploy start"

### DISCUSSION THREAD

**Discussion by jhgihub0** [forcedotcom/cli#2943] [Ideas]
Provide an easy way to view the zip file that is generated when running `sf project deploy start`

See https://github.com/forcedotcom/cli/discussions/2942

**Comment by VivekMChawla** [comment]
This idea is being considered. 

See [this comment](https://github.com/forcedotcom/cli/discussions/2942#discussioncomment-9988907) for a detailed discussion of proposed outcomes.

**Comment by cristiand391** [comment]
triaging old discussions...

i'll close this since we already printing zip info under the `--verbose` flag for deploys:
https://github.com/forcedotcom/cli/tree/main/releasenotes/#2577-september-4-2024


Regarding finding the zip that's deployed, you can set the `SF_MDAPI_TEMP_DIR=<dir>` env var to keep the contents of the metadata to be deployed (recommended for debugging, you shouldn't use it in prod deployments as it might affect performance)
https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_dev_cli_env_variables.htm#sfdx_dev_cli_env_variables
